foodsaving/utils/tests/test_email_utils.py

```diff
@@ -22,19 +24,19 @@ def test_emailinvitation(self):
             group=self.group,
             invited_by=self.user
         )
-        email = email_utils.prepare_emailinvitation_email(invitation)
-        self.assertEqual(len(email.alternatives), 0)
+        email = foodsaving.invitations.emails.prepare_emailinvitation_email(invitation)
+        self.assertEqual(len(email.alternatives), 1)
         self.assertEqual(email.to[0], 'bla@bla.com')
         self.assertIn(self.group.name, email.body)
         self.assertIn(self.user.display_name, email.body)
         self.assertIn(str(invitation.token), email.body)
```

Comment:
I observed a funny bug:
MJML inserted a newline (`\n`) in the middle of the invitation link:

```code
======================================================================
FAIL: test_emailinvitation (foodsaving.utils.tests.test_email_utils.TestEmailUtils)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib/python3.5/unittest/case.py", line 59, in testPartExecutor
    yield
  File "/usr/lib/python3.5/unittest/case.py", line 601, in run
    testMethod()
  File "/home/lifejam/workspace/karrot/karrot-backend/foodsaving/utils/tests/test_email_utils.py", line 31, in test_emailinvitation
    self.assertIn(str(invitation.token), email.body)
  File "/usr/lib/python3.5/unittest/case.py", line 1080, in assertIn
    self.fail(self._formatMessage(msg, standardMsg))
  File "/usr/lib/python3.5/unittest/case.py", line 666, in fail
    raise self.failureException(msg)
AssertionError: '7a4d2eff-ae15-47c7-acf6-33d16c5ef9fe' not found in '# You\'re invited to join Steven Collins!  \n  \nTyler Cruz invited you to join the "Steven Collins" community on karrot.world!\n\nIf you\'d like to join, please click the link below:  \n  \n[Follow the invitation][1]  \n  \n[][2]  \n  \n[Join our Facebook Group!][3] • made with love by [foodsaving worldwide][4]\n\n   [1]: http://localhost:8080/#/signup?invite=7a4d2eff-\nae15-47c7-acf6-33d16c5ef9fe&email=bla@bla.com\n\n   [2]: http://localhost:8080\n\n   [3]: https://www.facebook.com/groups/foodsaving.worldwide/\n\n   [4]: https://foodsaving.world\n\n'

----------------------------------------------------------------------
```

Reply:
That's probably the `html2text` library. If you switch the test to assert in HTML directly, they should be more reliable.

Reply:
That's right, `html2text` was causing the problem. It's fixed now.